src/viking.js

```diff
@@ -1,11 +1,124 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength){
+        this.health = health;
+        this.strength = strength;
+    
+    }
+    attack(){
+        return this.strength        
+    }
+    receiveDamage(damage){
+        this.health -= damage;
+        
+    }
+
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength){
+        super(health, strength);
+        this.name = name    
+    }
+
+    receiveDamage(damage){
+
+    super.receiveDamage(damage);
+    
+    if(this.health > 0){     
+
+    return `${this.name} has received ${damage} points of damage`
+    
+    }else if (this.health <= 0 ){
+
+        return `${this.name} has died in act of combat`
+        
+    }
+}
+battleCry(){
+
+    return "Odin Owns You All!"
+
+}
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier{
+    receiveDamage(damage){
+
+        super.receiveDamage(damage)
+
+        if(this.health > 0){
+            return `A Saxon has received ${damage} points of damage` ;
+        } else if(this.health <= 0){
+            return `A Saxon has died in combat`;
+        }
+    }
+}
+
 
 // War
-class War {}
+class War {
+    vikingArmy = [];
+    saxonArmy = [];
```

Comment:
Here I would place vikingArmy and and saxonArmy inside a `constructor()` and call it `this.vikingArmy` for example